src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {
+  Popover as CorePopover,
+  PopoverProps as CorePopoverProps,
+} from 'wix-ui-core/popover';
+
+import { TPAComponentProps } from '../../types';
+import { generateKey } from '../../common/random';
+import { KEYS } from '../../common/keyCodes';
+
+import { TriggerAction } from './constants';
+import { st, classes } from './Popover.st.css';
+
+export interface PopoverProps
+  extends TPAComponentProps,
+    Omit<CorePopoverProps, 'shown'> {
+  triggerAction?: TriggerAction;
+  wiredToSiteColors?: boolean;
+}
+
+export const Popover: React.FC<PopoverProps> & {
+  Element?: React.FC;
+  Content?: React.FC;
+} = props => {
+  const {
+    triggerAction,
+    wiredToSiteColors,
+    children,
+    className,
+    ...rest
+  } = props;
+  const [isShown, setIsShown] = React.useState<boolean>(false);
+  const [id] = React.useState(generateKey('popover'));
+
+  const [Element, Content] = React.Children.toArray(
+    children,
+  ) as React.ReactElement[];
+
+  return (
+    <CorePopover
+      id={id}
```

Comment:
So the `id` should actually be added to the tooltip's content container, in order for it to really describe the element

Reply:
Meaning this probably should move to the Popover.Content part